pkg/install/generate.go

```diff
@@ -42,6 +42,11 @@ const (
 	tryInstallOnceAnnotation              = "hive.openshift.io/try-install-once"
 	tryUninstallOnceAnnotation            = "hive.openshift.io/try-uninstall-once"
 	clusterDeploymentGenerationAnnotation = "hive.openshift.io/cluster-deployment-generation"
+	// installJobLabel is the label used for counting the number of install jobs in Hive
+	installJobLabel = "hive.openshift.io/install"
+
+	// uninstallJobLabel is the label used for counting the number of uninstall jobs in Hive
+	uninstallJobLabel = "hive.openshift.io/uninstall"
```

Comment:
Lets expose these by making first char upper case, and then reference them in the metrics package, so we have only one definition of them floating around.